core/transcoder.go

```diff
@@ -91,6 +150,28 @@ func (nv *NvidiaTranscoder) Stop() {
 	nv.session.StopTranscoder()
 }
 
+func runTranscodeLoop(gpu string, stack *segStack, workDir string) {
```

Comment:
Can we use the `gpu` field in `segStack` and omit the extra `gpu` param here?

Reply:
Fixed in 26c1f19b46bb9546bdbce22cbe3f890d957e0310

(note to self, this will cause a broken build for the fixed-up version of a15544b314103d1fc56cdcbfc836dd47a84514aa and probably a merge error. will fix when rebasing)